### PR TITLE
list 2 main YAML features: superset of JSON, yet human-readable

### PR DESCRIPTION
**Pull request by VladimirAlexiev**
***
<a href="https://pr-preview.s3.amazonaws.com/VladimirAlexiev/yaml-ld/pull/134.html" title="Last updated on Apr 9, 2024, 7:55 AM UTC (465d487)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/134/efd92b8...VladimirAlexiev:465d487.html" title="Last updated on Apr 9, 2024, 7:55 AM UTC (465d487)">Diff</a>

### CODE REVIEW

**Comment by VladimirAlexiev** [comment]
The precise preview link is https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/134/efd92b8...VladimirAlexiev:67f79de.html#json-vs-yaml .
Wow @gkellogg this is immensely useful!

File: spec/index.html [hunk 1]
```diff
@@ -596,7 +596,9 @@ <h2>Basic Concepts</h2>
     <section id="json-vs-yaml">
       <h3>JSON vs YAML comparison</h3>
 
-      <p>YAML is more flexible than JSON, as illustrated by comparison table below.</p>
+      <p>YAML is a superset of JSON, i.e. every valid JSON document is also a valid YAML document.
+        But YAML offers a number of extra features, chief amongst which is human readability due to the ability to use minimal punctuation.
+        YAML is more flexible than JSON, as illustrated by the comparison table below.</p>
```
Comment (anatoly-scherbakov) [inline spec/index.html:601]:
```suggestion
        But YAML offers a number of extra features, chief amongst which is human readability due to the ability to use minimal punctuation.</p>
        <p>YAML is more flexible than JSON, as illustrated by the comparison table below.</p>
```

Perhaps these could be on separate lines.

File: spec/data/json-vs-yaml.yaml [hunk 2]
```diff
@@ -91,10 +91,26 @@ $reverse:
...
+      yaml:
+        prov:value: yes
+        prov:wasDerivedFrom: YAML#chapter-7-flow-style-productions
+
+    - rdfs:label: Punctuation-free strings, arrays, objects
+      $type: features
+      json: no
+      yaml:
+        prov:value: yes
+        prov:wasDerivedFrom: YAML#chapter-8-block-style-productions
```
Comment (anatoly-scherbakov) [inline spec/data/json-vs-yaml.yaml:110]: Great addition! I did not think of highlighting this separation of flow vs block syntax which YAML supports.

**Review by anatoly-scherbakov — APPROVED** [review]
Approving; but I do not think we can merge now, can we? We should wait till WG is re-chartered to continue work on the spec.

**Comment by gkellogg** [comment]
The JSON-LD WG discussed this and decided that work can proceed on the editor's draft, so this can be merged.

File: spec/index.html [hunk 3]
```diff
@@ -597,8 +597,8 @@ <h2>Basic Concepts</h2>
       <h3>JSON vs YAML comparison</h3>
 
       <p>YAML is a superset of JSON, i.e. every valid JSON document is also a valid YAML document.
```
Comment (TallTed) [inline spec/index.html:599]:
```suggestion
      <p>YAML is a superset of JSON, i.e., every valid JSON document is also a valid YAML document.
```

File: spec/index.html [hunk 4]
```diff
@@ -597,8 +597,8 @@ <h2>Basic Concepts</h2>
...
-        But YAML offers a number of extra features, chief amongst which is human readability due to the ability to use minimal punctuation.
-        YAML is more flexible than JSON, as illustrated by the comparison table below.</p>
+        But YAML offers a number of extra features, chief amongst which is human readability due to the ability to use minimal punctuation.</p>
```
Comment (TallTed) [inline spec/index.html:600]:
```suggestion
        YAML also offers a number of extra features, chief amongst which is improved human readability due to the ability to use minimal punctuation.</p>
```

**Review by TallTed — CHANGES_REQUESTED** [review]
minor language and punctuation

**Comment by anatoly-scherbakov** [comment]
@VladimirAlexiev I apologize: I reviewed, and committed, changes by @TallTed did not noticing that they were to a PR that is not mine.

**Review by TallTed — APPROVED** [review]
lgtm

**Review by gkellogg — APPROVED** [review]

File: spec/index.html [hunk 5]
```diff
@@ -596,7 +596,9 @@ <h2>Basic Concepts</h2>
     <section id="json-vs-yaml">
       <h3>JSON vs YAML comparison</h3>
 
-      <p>YAML is more flexible than JSON, as illustrated by comparison table below.</p>
+      <p>YAML is a superset of JSON, i.e., every valid JSON document is also a valid YAML document.
+        YAML also offers a number of extra features, chief amongst which is improved human readability due to the ability to use minimal punctuation.</p>
+        <p>YAML is more flexible than JSON, as illustrated by the comparison table below.</p>
```
Comment (davidlehn) [inline spec/index.html:601]: Fix indent.
```suggestion
      <p>YAML is more flexible than JSON, as illustrated by the comparison table below.</p>
```

**Comment by anatoly-scherbakov** [comment]
@VladimirAlexiev would you be open to merging this PR, or maybe you wish to add more changes?

**Comment by anatoly-scherbakov** [comment]
I think I will boldly go and merge :)

**Pull request merged**